.travis.yml

```diff
@@ -15,9 +15,21 @@ cache:
   directories:
     - $HOME/.composer/cache
 
+addons:
+  ssh_known_hosts: $deploy_host
+
+before_install:
+  - composer self-update
+  - openssl aes-256-cbc -K $encrypted_4d43efaba550_key -iv $encrypted_4d43efaba550_iv -in deploy_rsa.enc -out /tmp/deploy_rsa -d
```

Comment:
please use more appropriate var name e.g. $rsa_key and $rsa_iv

Reply:
These variables are generated by Travis encryption process and I have no access to their values so I can't recreate them